<launch>
  <arg name="pose_topic"          default="/amcl_pose" /> 

  <!-- Launch your Python script with a specified pose_topic parameter -->
  <node name="compute_localization_error" pkg="fiducial_navigation" type="compute_localization_error.py" output="screen">
    <param name="pose_topic" type="string" value="$(arg pose_topic)"/>
  </node>
</launch>
